src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -180,23 +191,58 @@ export class HaConfigDeviceDashboard extends SubscribeMixin(LitElement) {
     },
   ]);
 
-  firstUpdated() {
-    this._filters = {
-      "ha-filter-states": {
-        value: [],
-        items: undefined,
-      },
-    };
-    this._setFiltersFromUrl();
+  willUpdate(changedProps) {
+    if (!this.hasUpdated) {
+      this._setFiltersFromUrl();
+    }
+    if (changedProps.has("_filters") || changedProps.has("_filter")) {
+      this._updateUrl();
```

Comment:
We should remove the usage of url (except for config entry as before)

Reply:
Oops, yeah this is leftover code that should have been removed